src/Proxy/SernosUsedOnInvoiceReportProxy.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace Linn.Products.Proxy
+{
+    using System;
+    using System.Data;
+    using Linn.Products.Domain.Linnapps.RemoteServices;
+
+    public class SernosUsedOnInvoiceReportProxy : ISernosUsedOnInvoiceDatabaseService
+    {
+        private readonly IDatabaseService databaseService;
+
+        public SernosUsedOnInvoiceReportProxy(IDatabaseService databaseService)
+        {
+            this.databaseService = databaseService;
+        }
+
```

Comment:
@pyotrritchie is this better? Moved the database stuff out of here so it can be injected,